### PR TITLE
Added Code of Conduct to Docs

### LINKED ISSUE #381: Fix Grammatical issues in Code of Conduct and add it in docs
🐞 **Problem**

The language in the COC is flawed.

```
 We will take action when members of our community violate this code such as contacting einsteinpy.project@gmail.com (all emails sent to this address will be treated with the strictest confidence) or talking privately with the person.
```

🎯 **Goal**

Fix it,

And add the COC to the documentation as well.

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by ShamanthNyk**
Fixes #381

### CODE REVIEW

**Comment by ritzvik** [comment]
@ShamanthNyk , It's your first PR, so cheers!!

See that the CI for docs is failing. Kindly open the logs, and try to rectify it.

**Comment by ritzvik** [comment]
One more thing @ShamanthNyk . Next time onward, please try to open a new branch, for every PR you propose. Working from `master` is never a good idea.

**Comment by ShamanthNyk** [comment]
@ritzvik I am sorry, I will consider that for my next PR's

File: docs/source/codeofconduct.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    • We pledge to welcome questions and answer them respectfully, paying particular attention to those new to the community. We pledge to provide respectful criticisms and feedback in forums, especially in discussion threads resulting from code contributions.
+    • We pledge to be conscientious of the perceptions of the wider community and to respond to criticism respectfully. We will strive to model behaviors that encourage productive debate and disagreement, both within our community and where we are criticized. We will treat those outside our community with the same respect as people within our community.
+    • We pledge to help the entire community follow the code of conduct, and to not remain silent when we see violations of the code of conduct. Action will be taken against the members who violate the code which include the following-contacting einsteinpy.project@gmail.com (all emails sent to this address will be treated with the strictest confidence),talking privately with the person.
```
Comment (shreyasbapat) [inline docs/source/codeofconduct.rst]: There are some grammatical mistakes in this :) 

Do a `git pull` before addressing them. Also we need to change that in the main COC. See #381  for more details

**Comment by ritzvik** [comment]
@ShamanthNyk you have made a new PR, although it was not necessary. So close this one?

**Comment by shreyasbapat** [comment]
Closed for #396

**Pull request closed without merging**